nservicebus/upgrades/gateway-3to4.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Gateway Upgrade Version 3 to 4
+summary: How to upgrade the Gateway from version 3 to 4.
+component: Gateway
+reviewed: 2022-09-22
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 7
+ - 8
+---
+
+## Migration from BinaryFormatter
+
```

Comment:
WARNING: When migrating from the `BinaryFormatter` ensure that all endpoints on all sites are updated to use the newly selected databus deserializer.